config/routes.rb

```diff
@@ -1,4 +1,5 @@
 Rails.application.routes.draw do
+  devise_for :users
   resources :users, only: [:index, :show] do
     resources :posts, only: [:index,:new,:create,:show]
```

Comment:
- [ ] [OPTIONAL]
Generally, if a user not signed in, the user should be redirected to the login page, however, when enter the app it always assumes that the user is signed in. Let's endeavor to improve upon this aspect.

**Root URL blank:**
![image](https://github.com/wolterbwambale/blog_app/assets/12745474/882ac778-c404-41f1-88b6-37b756c0cd36)


We can improve upon this aspect by modifying the routes to be similar to:
```ruby
Rails.application.routes.draw do
    devise_for :users, path: "", path_names: {sign_in:"login", sign_out:"logout"}, controllers: {
      sessions: 'users/sessions', 
      registrations: 'users/registrations'
}

root "users#index"
resources :users, only: [:index, :show] do
  resources :posts, only: [:index, :show, :new, :create] do
    resources :comments, only: :create 
    resources :likes, only: [:create, :destroy]
 end
end
end
```